[debug] [2023-06-13T08:01:38.425Z] ----------------------------------------------------------------------
[debug] [2023-06-13T08:01:38.428Z] Command:       /usr/bin/node /usr/bin/firebase init
[debug] [2023-06-13T08:01:38.429Z] CLI Version:   12.3.1
[debug] [2023-06-13T08:01:38.429Z] Platform:      linux
[debug] [2023-06-13T08:01:38.429Z] Node Version:  v20.2.0
[debug] [2023-06-13T08:01:38.430Z] Time:          Tue Jun 13 2023 11:01:38 GMT+0300 (East Africa Time)
[debug] [2023-06-13T08:01:38.430Z] ----------------------------------------------------------------------
[debug] 
[debug] [2023-06-13T08:01:38.435Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2023-06-13T08:01:38.495Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2023-06-13T08:01:38.544Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (/usr/lib/node_modules/firebase-tools/node_modules/google-auth-library/build/src/auth/googleauth.js:183:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async GoogleAuth.getClient (/usr/lib/node_modules/firebase-tools/node_modules/google-auth-library/build/src/auth/googleauth.js:565:17)
    at async GoogleAuth.getAccessToken (/usr/lib/node_modules/firebase-tools/node_modules/google-auth-library/build/src/auth/googleauth.js:587:24)
    at async autoAuth (/usr/lib/node_modules/firebase-tools/lib/requireAuth.js:24:19)
    at async Object.requireAuth [as fn] (/usr/lib/node_modules/firebase-tools/lib/requireAuth.js:48:20)
    at async /usr/lib/node_modules/firebase-tools/lib/command.js:214:17
